### DISCUSSION TITLE
Pull a model from Ollama.com // multiple ollama servers

### DISCUSSION THREAD

**Discussion by UberMetroid** [open-webui/open-webui#5273] [Ideas]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I download a new LLM from ollama.com because I have multiple ollama servers and have to download the model for each server.

**Describe the solution you'd like**
I would like an option to match llms across ollama servers so I dont have to manually check each ollama server to make sure the llm is there.

**Describe alternatives you've considered**
Currently you have to download the llm for one server, then switch to the next and download it again.

**Comment by mdlmarkham** [comment]
One additional twist - some of my instances of Ollama are on less powerful machines... and I don't want to download big models on them. It would be ideal if OpenWebUI could:
- Route requests to the most capable model that is available
- Use Wake-On-Lan to bring machines online as needed
